exchanges/bybit/bybit_wrapper.go

```diff
@@ -0,0 +1,1494 @@
+			Date:           resp.Time.Time(),
+			LastUpdated:    resp.UpdateTime.Time(),
+		}, nil
+	case asset.CoinMarginedFutures:
+		resp, err := by.GetActiveRealtimeCoinOrders(ctx, pair, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("invalid order's count found")
+		}
+
+		return order.Detail{
+			Amount:         resp[0].Qty,
+			Exchange:       by.Name,
+			ID:             resp[0].OrderID,
+			ClientOrderID:  resp[0].OrderLinkID,
+			Side:           order.Side(resp[0].Side),
+			Type:           order.Type(resp[0].OrderType),
+			Pair:           pair,
+			Cost:           resp[0].CumulativeQty,
+			AssetType:      assetType,
+			Status:         order.Status(resp[0].OrderStatus),
+			Price:          resp[0].Price,
+			ExecutedAmount: resp[0].Qty - resp[0].LeavesQty,
+			Date:           resp[0].CreatedAt,
+			LastUpdated:    resp[0].UpdatedAt,
+		}, nil
+
+	case asset.USDTMarginedFutures:
+		resp, err := by.GetActiveUSDTRealtimeOrders(ctx, pair, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("invalid order's count found")
+		}
+
+		return order.Detail{
+			Amount:         resp[0].Qty,
+			Exchange:       by.Name,
+			ID:             resp[0].OrderID,
+			ClientOrderID:  resp[0].OrderLinkID,
+			Side:           order.Side(resp[0].Side),
+			Type:           order.Type(resp[0].OrderType),
+			Pair:           pair,
+			Cost:           resp[0].CumulativeQty,
+			AssetType:      assetType,
+			Status:         order.Status(resp[0].OrderStatus),
+			Price:          resp[0].Price,
+			ExecutedAmount: resp[0].Qty - resp[0].LeavesQty,
+			Date:           resp[0].CreatedAt,
+			LastUpdated:    resp[0].UpdatedAt,
+		}, nil
+
+	case asset.Futures:
+		resp, err := by.GetActiveRealtimeOrders(ctx, pair, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("invalid order's count found")
+		}
+
+		return order.Detail{
+			Amount:         resp[0].Qty,
+			Exchange:       by.Name,
+			ID:             resp[0].OrderID,
+			ClientOrderID:  resp[0].OrderLinkID,
+			Side:           order.Side(resp[0].Side),
+			Type:           order.Type(resp[0].OrderType),
+			Pair:           pair,
+			Cost:           resp[0].CumulativeQty,
+			AssetType:      assetType,
+			Status:         order.Status(resp[0].OrderStatus),
+			Price:          resp[0].Price,
+			ExecutedAmount: resp[0].Qty - resp[0].LeavesQty,
+			Date:           resp[0].CreatedAt,
+			LastUpdated:    resp[0].UpdatedAt,
+		}, nil
+
+	default:
+		return order.Detail{}, fmt.Errorf("assetType not supported: %v", assetType)
+	}
+}
+
+// GetDepositAddress returns a deposit address for a specified currency
+func (by *Bybit) GetDepositAddress(ctx context.Context, cryptocurrency currency.Code, _, chain string) (*deposit.Address, error) {
+	return nil, common.ErrNotYetImplemented
```

Comment:
These need to be implemented: https://bybit-exchange.github.io/docs/account_asset/#t-transfer_api and below

Reply:
@shazbert 
I didn't you. These APIs are for internal transfers between different asset types in Bybit account.
I don't see any API related to GetDepositAddress or WithdrawCryptocurrencyFunds etc.

Reply:
@Jdpurohit Holy moly you are correct, my apologies; will resolve.